### ISSUE TITLE
Google Plus Games Don't work while this is enabled

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bliu886/scriptno#57]
```
What steps will reproduce the problem?
1. log into google plus
2. open Monster World

What is the expected output? What do you see instead?
It is expected that the game loads, instead the flash loading screen stays 
forever

I have "allowed" and/or "trusted" all resources on the page, even when "Allow 
All Blocked for session" is used, the page still hangs unless I completely 
disable ScriptNo.  The only resource still blocked is a webbug related to 
accounts.google.com/o/oath2 which I'm guessing is required for these games to 
work.  This probably should not be considered a webbug, but I don't know how it 
is used elsewhere (so maybe it is and this is a bogus issue).  But in general 
if removing a webbug breaks a page, it probably should not be considered a 
webbug.


```

Original issue reported on code.google.com by `Jwadam...@gmail.com` on 11 Feb 2012 at 12:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 15 Feb 2012 at 7:14

**Comment by GoogleCodeExporter** [comment]
```
Slideshow on Huffington Post here does not work for the same reason - a 
<WEBBUG> is being blocked and cannot be unblocked. As a result, images do not 
show when the slideshow is run nor when the slideshow advance arrows are 
clicked manually. Some images show, most do not. The first image will show but 
not subsequent images, despite allowing ALL resource on the page except the 
<WEBBUG>. This is broken and should be fixed.

http://www.huffingtonpost.com/2012/02/15/babes-at-the-new-museum-triennial_n_127
9001.html?ir=New+York#s698517

```

Original comment by `richards...@gmail.com` on 15 Feb 2012 at 8:11

**Comment by GoogleCodeExporter** [comment]
```
I did explicitly/manually add the webbug to my whitlelist, which solved the 
issue. But it was not possible via the toolbar menu.

It would also be nice (perhaps a new issue should be opend) if I could 
whitelist by page all resources on a page, not an entire domain nor host, nor 
resources from 3rd party domains when on a different site. Effectivly I want to 
be able to disable scriptno *completely* for a given page or wildcard url.
```

Original comment by `Jwadam...@gmail.com` on 15 Feb 2012 at 9:41
